### PR TITLE
SQR3-225: Queue Abstraction for Mainflux & RabbitMQ Support

### PR DESCRIPTION
**Pull request by SaraGhlm**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+variables:
```
Comment (HolyKau) [inline .gitlab-ci.yml]: We don't need the gitlab CI in github

File: docker/README.md [hunk 2]
```diff
@@ -22,3 +22,5 @@ docker-compose -f docker/addons/<path>/docker-compose.yml  up
 ```
 
 To pull docker images from a specific release you need to change the value of `MF_RELEASE_TAG` in `.env` before running these commands.
+
+Each messaging system has a separate docker-compose.yml file. To specify a messaging system update `MF_QUEUE_SYSTEM` in `.env`, and execure `run.sh` file to run the respective docker-compose.yml file.
```
Comment (HolyKau) [inline docker/README.md]: Typo "execure"

File: docker/addons/cassandra-writer/docker-compose.yml [hunk 3]
```diff
@@ -36,6 +36,22 @@ services:
...
+      MF_RABBITMQ_TLS_CERTIFICATE: ${MF_RABBITMQ_TLS_CERTIFICATE}
+      MF_RABBITMQ_TLS_KEY: ${MF_RABBITMQ_TLS_KEY}
+      MF_RABBITMQ_TLS_CA: ${MF_RABBITMQ_TLS_CA}
+      MF_RABBITMQ_DURABLE: ${MF_RABBITMQ_DURABLE}
+      MF_RABBITMQ_TTL: ${MF_RABBITMQ_TTL}
+      MF_RABBITMQ_PRIORITY: ${MF_RABBITMQ_PRIORITY}
+      MF_RABBITMQ_CONTENT_TYPE: ${MF_RABBITMQ_CONTENT_TYPE}
+      MF_RABBITMQ_SUB_SYSTEM: ${MF_RABBITMQ_SUB_SYSTEM}
+      MF_RABBITMQ_SEVERITY: ${MF_RABBITMQ_SEVERITY}
+      MF_RABBITMQ_QUEUES: ${MF_RABBITMQ_QUEUES}s
```
Comment (HolyKau) [inline docker/addons/cassandra-writer/docker-compose.yml]: Extra s

File: docker/README.md (hunk 2 above)
Comment (SaraGhlm) [reply docker/README.md]: fixed

File: docker/addons/cassandra-writer/docker-compose.yml (hunk 3 above)
Comment (SaraGhlm) [reply docker/addons/cassandra-writer/docker-compose.yml]: fixed

File: .gitlab-ci.yml (hunk 1 above)
Comment (SaraGhlm) [reply .gitlab-ci.yml]: removed

**Pull request closed without merging**